packages/reactor-api/src/subgraphs/drive/index.ts

```diff
@@ -1,4 +1,392 @@
-import { typeDefs } from "./type-defs";
-import { resolvers } from "./resolvers";
+import { GraphQLResolverMap } from "@apollo/subgraph/dist/schema-helper";
+import {
+  generateUUID,
+  ListenerRevision,
+  PullResponderTransmitter,
+  StrandUpdateGraphQL,
+} from "document-drive";
+import {
+  actions,
+  DocumentDriveAction,
+  FileNode,
+  Listener,
+  ListenerFilter,
+  TransmitterType,
+} from "document-model-libs/document-drive";
+import { Asset } from "document-model-libs/real-world-assets";
+import { BaseAction, Operation } from "document-model/document";
+import {
+  DocumentModelInput,
+  DocumentModelState,
+} from "document-model/document-model";
+import { gql } from "graphql-tag";
+import { Context } from "../types";
+import { Subgraph } from "../base";
 
-export { typeDefs, resolvers };
+export class DriveSubgraph extends Subgraph {
+  name = "drive/:driveId";
+  typeDefs = gql`
```

Comment:
Let's define the drive typedefs and resolvers in different files. It will also make it easier to unit test the resolvers without having to instantiate a DriveSubgraph instance.